### PR TITLE
add docs about syncbn && weight_norm && remove_weight_norm

### PR DESCRIPTION
**Pull request by ceci3**
add docs about syncbn && weight_norm && remove_weight_norm

### CODE REVIEW

**Comment by ceci3** [comment]
image

**Comment by ceci3** [comment]
SyncBatchNorm:

屏幕快照 2020-08-21 上午10 28 14
屏幕快照 2020-08-21 上午10 28 19

File: doc/fluid/api_cn/nn_cn/SyncBatchNorm_cn.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.. _cn_api_fluid_dygraph_SyncBatchNorm:
+
+SyncBatchNorm
+-------------------------------
+
+.. py:class:: paddle.fluid.dygraph.SyncBatchNorm(num_features, epsilon=1e-5, momentum=0.9, track_running_stats=True, weight_attr=None, bias_attr=None, data_format='NCHW', name=None)
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/SyncBatchNorm_cn.rst]: paddle.nn.SyncBatchNorm

File: doc/fluid/api_cn/nn_cn/SyncBatchNorm_cn.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+- :math:`\gamma` : 可训练的比例参数
+- :math:`\beta` : 可训练的偏差参数
+
+参数：
+    - **num_features** (int) - 指明输入 ``Tensor`` 的通道数量。
+    - **epsilon** (float, 可选) - 为了数值稳定加在分母上的值。默认值：1e-05。
+    - **momentum** (float, 可选) - 此值用于计算 ``moving_mean`` 和 ``moving_var`` 。默认值：0.9。更新公式如上所示。
+    - **weight_attr** (ParamAttr|bool, 可选) - 指定权重参数属性的对象。如果设置为 ``False`` ，则表示本层没有可训练的权重参数。默认值为None，表示使用默认的权重参数属性。具体用法请参见 :ref:`cn_api_fluid_ParamAttr` 。
+    - **bias_attr** (ParamAttr|bool, 可选) - 指定偏置参数属性的对象。如果设置为 ``False`` ，则表示本层没有可训练的偏置参数。默认值为None，表示使用默认的偏置参数属性。具体用法请参见 :ref:`cn_api_fluid_ParamAttr` 。
+    - **track_running_stats** （bool, 可选）- 是否计算全局均值和方差。
```
Comment (TCChenlong) [inline doc/fluid/api_cn/nn_cn/SyncBatchNorm_cn.rst]: 默认值：True，表示xxx

File: doc/fluid/api_cn/nn_cn/SyncBatchNorm_cn.rst (hunk 1 above)
Comment (ceci3) [reply doc/fluid/api_cn/nn_cn/SyncBatchNorm_cn.rst]: done, thanks

File: doc/fluid/api_cn/nn_cn/SyncBatchNorm_cn.rst (hunk 2 above)
Comment (ceci3) [reply doc/fluid/api_cn/nn_cn/SyncBatchNorm_cn.rst]: done, thanks

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**